python/tutorials/06-fused-attention.py

```diff
@@ -200,10 +200,10 @@ def _bwd_kernel(
             q = tl.load(q_ptrs)
             # recompute p = softmax(qk, dim=-1).T
             if CAUSAL:
-                qk = tl.dot(q, tl.trans(k), out_dtype=tl.float32)
-                qk = tl.where(offs_m_curr[:, None] >= (offs_n[None, :]), qk, float("-inf"))
+                qk = tl.where(P_SEQ + offs_m_curr[:, None] >= (offs_n[None, :]), float(0.), float("-inf"))
             else:
-                qk = tl.dot(q, tl.trans(k), out_dtype=tl.float32)
+                qk = tl.zeros([BLOCK_M, BLOCK_N], dtype=tl.float32)
+            qk += tl.dot(q, tl.trans(k))
```

Comment:
Nitpicky comment, but do you think it'd be cleaner to do it how the fwd kernel does it at line 91 above?

Reply:
Actually, I like it in fwd better. But intention of this PR is to align our fork with upstream, to make IFUs and stuff easier, so I just copied what we have in upstream.

Reply:
Not sure, if there any difference in term of performance, though.
We should check it, I think.

I believe we do not use this version for benchmarking, so I did not check it yet.